crates/bevy_ecs/src/query/state.rs

```diff
@@ -398,6 +409,29 @@ impl<D: QueryData, F: QueryFilter> QueryState<D, F> {
         }
     }
 
+    // TODO: Better explanation, I'm not a fan of the term "reffed".
+    /// Gets the query result in "reffed" form (whatever can go in the [`Ref`] smart pointer - goes into it,
```

Comment:
Perhaps something like "Gets a read-only smart pointer with change-detection information to the query results for the given `World` and `Entity`."